lib/grape/formatters/active_model_serializers.rb

```diff
@@ -0,0 +1,16 @@
+# A grape response formatter that can be used as 'formatter :json, Grape::Formatters::ActiveModelSerializers'
+#
+# Serializer options can be passed as a hash from your grape endpoint using env[:active_model_serializer_options],
+# or better yet user the render helper in Grape::Helpers::ActiveModelSerializers
+module Grape
+  module Formatters
+    module ActiveModelSerializers
+      def self.call(resource, env)
+        return resource.to_json unless resource.is_a?(Array) || resource.respond_to?(:model_name) || resource.class.respond_to?(:model_name)
```

Comment:
just thinking out loud: might be easier to read as

```ruby
if resource.is_a...
  resource.to_json
else
  ..new(..).to_json
end
```

Naive question: ActiveModelSerializers should handle Arrays and non-models just fine. See the logic in [ActionController::Serialization](https://github.com/rails-api/active_model_serializers/blob/5d0cc5990a18c9371441bad90cd8c50f642640fd/lib/action_controller/serialization.rb#L29-L55).   Any reason you're checking for an Array or `model_name`?